test/metabase/query_processor/middleware/annotate_test.clj

```diff
@@ -505,6 +505,45 @@
       :cols
       second))
 
+(expect
```

Comment:
can you write new tests using `deftest` instead of `expect`? Our long-term plan is to be pure `clojure.test`. @flyingmachine and I have been slowly rewriting existing tests so it's a long-term project but we shouldn't add new ones using the old pattern

it's way easier to run tests from the REPL with `clojure.test`